client/src/components/FlagDetailsPage.jsx

```diff
@@ -121,7 +120,7 @@ const FlagDetailsPage = () => {
         <EditFlagForm setIsEditing={setIsEditing} />
       )}
       {exptData &&
-        exptData.map((expt) => {
+        exptData.sort((a, b) => Number(b.id) - Number(a.id)).map((expt) => {
```

Comment:
I think it might be more efficient if the sorting is done in the SQL query? Right now if you check in the `constants` file I have it as `ORDER BY date_started DESC` so you can just change that to `id`.

Reply:
Good idea, just implemented that.